specification/ai/OpenAI.Assistants/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/openai/assistants"
+    default: "sdk/openai"
```

Comment:
Fix this as well.

SDK is in `sdk/openai` folder, not `sdk/openai/assistants` folder.
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/openai
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/openai
https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/openai

Reply:
so this config (`service-dir`) does not impact current generated code?

Reply:
If generate in SDK repo, it seems not impact (as the direct that runs tsp-client is the folder).

However, when SDK automation runs on the spec repo, it affects.